### DISCUSSION TITLE
Feature Request: Add Google Lens by Area to Context Menu

### DISCUSSION THREAD

**Discussion by uses0ap** [Alex313031/thorium#899] [Ideas]
Request to add Google Lens 'Search By Area' feature to the right-click context menu:

![image](https://github.com/user-attachments/assets/36118f71-b6d3-4711-8088-0ecdab091ae9)
![image](https://github.com/user-attachments/assets/bf7f120a-7ee1-482a-8937-d5dd6089a07e)
![image](https://github.com/user-attachments/assets/cf493760-cb4a-481d-99b3-748c7255282c)
![image](https://github.com/user-attachments/assets/806d4092-72ec-4c96-a351-fb6ee17ead43)

**Comment by gz83** [comment]
I believe a similar menu is already included.
